### ISSUE TITLE
Esperando con interés actualizar más contenido

### ISSUE THREAD

**Issue by domainbf** [joanlajarin/country-page#1]
Hello, the program is good, but the adaptation to the mobile phone is not perfect.

In addition, Taiwan belongs to China and it should not be a country alone.

**Comment by joanlajarin** [comment]
Hello Li. Thank you for your feedback! Regarding Taiwan, I retrieved the information exactly as defined in the API that I used: https://restcountries.com/

**Comment by domainbf** [comment]
Yes, I have seen many people use this api, but there is something wrong with its information. Thank you. I have submitted feedback on this mistake in Taiwan. I hope they can correct it.
